<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CameraButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>촬영</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Led_RED">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Led_Off">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>끄기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buzzer_Off">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>끄기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buzzer_On">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>켜기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Buzzer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Project/Camera.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Project/LED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/Camera.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/LED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/Buzzer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Led_BLUE">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/hygrometer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTemper">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>작동</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LN_T">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LN_H">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
